data/ecosystems/r/radixcharts.toml

```diff
@@ -0,0 +1,10 @@
+# Ecosystem Level Information
+title = "RadixCharts"
+
+sub_ecosystems = []
+
+github_organizations = ["https://github.com/NellySayon"]
```

Comment:
this link corresponds to a github user account, not an org, so we can't use it, please remove

Reply:
fixed it